general/models.py

```diff
@@ -40,48 +38,8 @@ class AkismetSpam(SocialModel):
     spam = models.TextField()
     created = models.DateTimeField(auto_now_add=True)
 
-class OrderedModel(models.Model):
-    order = models.PositiveIntegerField(editable=False)
-
-    def save(self, *args, **kwargs):
-        if not self.id:
-            try:
-                self.order = self.__class__.objects.all().order_by("-order")[0].order + 1
-            except IndexError:
-                self.order = 0
-        super().save(*args, **kwargs)
-
-    def change_order(self):
-        model_type_id = ContentType.objects.get_for_model(self.__class__).id
-        model_id = self.id
-        kwargs = {"direction": "up", "model_type_id": model_type_id, "model_id": model_id}
-        url_up = reverse("admin-move", kwargs=kwargs)
-        kwargs["direction"] = "down"
-        url_down = reverse("admin-move", kwargs=kwargs)
-        return mark_safe(f'<a href="{url_up}">up</a> | <a href="{url_down}">down</a>')
-    change_order.short_description = 'Move'
-    change_order.admin_order_field = 'order'
-                
-    @staticmethod
-    def move(direction, model_type_id, model_id):
-        try:
-            ModelClass = ContentType.objects.get(id=model_type_id).model_class()
-
-            current_model = ModelClass.objects.get(id=model_id)
-            
-            if direction == "down":
-                swap_model = ModelClass.objects.filter(order__gt=current_model.order).order_by("order")[0]
-            elif direction == "up":
-                swap_model = ModelClass.objects.filter(order__lt=current_model.order).order_by("-order")[0]
-            
-            current_model.order, swap_model.order = swap_model.order, current_model.order
-
-            current_model.save()
-            swap_model.save()
-        except IndexError:
-            pass
-        except ModelClass.DoesNotExist:
-            pass
+class OrderedModel(SortableMixin):
+    order = models.PositiveIntegerField(default=0, editable=False)
```

Comment:
The migrations are necessary because of this change.

(I follow the example in [django-admin-sortable](https://github.com/jazzband/django-admin-sortable) in setting `default = 0`)